### PR TITLE
Sonar Fixes for squid:S4488 and fixed some spelling

### PR DESCRIPTION
**Pull request by warlockza**

### CODE REVIEW

File: src/main/java/org/cloudfoundry/promregator/PromregatorApplication.java [hunk 1]
```diff
@@ -218,7 +218,7 @@ public void forceGC() {
 	 * @return the unique identifier of the currently running instance
 	 */
 	@Bean
-	public UUID promregatorInstanceIdentifer() {
+	public UUID promregatorInstanceIdentifier() {
```
Comment (eaglerainbow) [inline src/main/java/org/cloudfoundry/promregator/PromregatorApplication.java]: What a luck that `@Autowired` also works on type basis.
(nothing to be changed here!)

File: src/main/java/org/cloudfoundry/promregator/PromregatorApplication.java [hunk 2]
```diff
@@ -167,7 +167,7 @@ public InternalMetrics internalMetrics() {
 	
 	@Value("${promregator.endpoint.threads:#{null}}")
 	@Deprecated
-	/**
+	/*
```
Comment (eaglerainbow) [inline src/main/java/org/cloudfoundry/promregator/PromregatorApplication.java]: I am a little unsure whether removing the javadoc aspect here is the right thing. In Eclipse, it's no longer shown what shall be used instead...

File: src/main/java/org/cloudfoundry/promregator/endpoint/AbstractMetricsEndpoint.java [hunk 3]
```diff
@@ -86,7 +86,7 @@
 
 	@Value("${promregator.endpoint.maxProcessingTime:#{null}}")
 	@Deprecated
-	/**
+	/*
```
Comment (eaglerainbow) [inline src/main/java/org/cloudfoundry/promregator/endpoint/AbstractMetricsEndpoint.java]: I am a little unsure whether removing the javadoc aspect here is the right thing. In Eclipse, it's no longer shown what shall be used instead...

File: src/main/java/org/cloudfoundry/promregator/PromregatorApplication.java (hunk 2 above)
Comment (warlockza) [reply src/main/java/org/cloudfoundry/promregator/PromregatorApplication.java]: Its either a javadoc or a comment.. If its a javadoc, it should be fully javadocced .. if its just a comment, we treat it as a comment. My understanding of the issue above is that its currently half javadoc, half comment.

so either
/*  
xxxx
*/

or 

/**
Javadockystuff
**/

File: src/main/java/org/cloudfoundry/promregator/PromregatorApplication.java [hunk 4]
```diff
@@ -167,7 +167,7 @@ public InternalMetrics internalMetrics() {
...
 	 * use threadPoolSize instead
```
Comment (eaglerainbow) [inline src/main/java/org/cloudfoundry/promregator/PromregatorApplication.java]:
```suggestion
	 * The number of threads of the scraping thread pool. The value is coming from the deprecated configuration option <pre>promregator.endpoint.threads</pre>. Use threadPoolSize instead
```

File: src/main/java/org/cloudfoundry/promregator/PromregatorApplication.java (hunk 2 above)
Comment (eaglerainbow) [inline src/main/java/org/cloudfoundry/promregator/PromregatorApplication.java]:
```suggestion
	/**
```
Then let's make it to a real JavaDoc

File: src/main/java/org/cloudfoundry/promregator/endpoint/AbstractMetricsEndpoint.java (hunk 3 above)
Comment (eaglerainbow) [inline src/main/java/org/cloudfoundry/promregator/endpoint/AbstractMetricsEndpoint.java]:
```suggestion
	/**
```

File: src/main/java/org/cloudfoundry/promregator/endpoint/AbstractMetricsEndpoint.java [hunk 5]
```diff
@@ -86,7 +86,7 @@
...
 	 * use maxProcessingTime instead
```
Comment (eaglerainbow) [inline src/main/java/org/cloudfoundry/promregator/endpoint/AbstractMetricsEndpoint.java]:
```suggestion
	 * The maximal processing time permitted for Scraping (in milliseconds). The value is deprecated as it originates from the deprecated configuration option <pre>promregator.endpoint.maxProcessingTime</pre>. Use maxProcessingTime instead.
```

Comment (warlockza) [reply src/main/java/org/cloudfoundry/promregator/endpoint/AbstractMetricsEndpoint.java]: So I worked out what a dangling Javadoc is now

Its where the javadoc is not completely above the property/method and its annotations.

Apparently, it needs to be before any annotations..... updated...

**Comment by eaglerainbow** [comment]
BTW: You are aware of https://sonarcloud.io/dashboard?id=promregator ?
Just managed to configure this today...

**Comment by warlockza** [comment]
Yes - I had a thorough peek at it . i saw you spent a good chunk of time cleaning stuff up.  I'm picking the low hanging non-invasive stuff until I am more familiar with the code base.

**Comment by eaglerainbow** [comment]
Sounds good - but: I feel a little ashamed that you would have to do the nasty task and be the "vacuum cleaner" for my sloppiness 😁 
So, please feel encourage to also "really touch" the coding once you feel comfortable doing so!

**Pull request merged**